src/styles/pages/roar-privacy-tos.scss

```diff
@@ -1,8 +1,23 @@
 .layout.privacy-tos-page {
+  header {
+    position: initial;
+  }
+
   .privacy-tos-section {
-    padding-top: 10rem;
-    h3 {
-      padding: 1rem 0;
+    padding: 0 5rem;
+    text-align: center;
```

Comment:
Move this inside the `h1` declaration so that you may remove the `text-align: left;` from the `h3` and `p` declarations

Reply:
changed syntax as instructed. Paragraph text all centered. Please double check.

Reply:
all worked.